lib/shoryuken/environment_loader.rb

```diff
@@ -99,7 +99,7 @@ def prefix_active_job_queue_names
     end
 
     def parse_queue(queue, weight = nil)
-      [weight.to_i, 1].max.times { Shoryuken.queues << queue }
+      Shoryuken.add_queue(queue, [weight.to_i, 1])
```

Comment:
You have not take the max of `[weight.to_i, 1]` but is it OK?
I think that `add_queue` takes `weight` which is defined numeric.